default.config.yml

```diff
@@ -98,37 +98,36 @@ drupalvm_cron_jobs: []
 # this variable is 'true'.
 configure_drush_aliases: true
 
+apache_vhosts_extra_parameters: |
+    <FilesMatch \.php$>
+      SetHandler "proxy:fcgi://{{ php_fpm_listen }}"
+    </FilesMatch>
```

Comment:
If you could just add a comment above the variable explaining this is to help configure the PHP-FPM connection to Apache... something like that would be adequate.